StripeConnect/StripeConnect/Source/Internal/Webview/ConnectWebView.swift

```diff
@@ -111,16 +116,31 @@ private extension ConnectWebView {
         urlOpener.openIfPossible(url)
     }
 
-    func showErrorAlert(for error: Error) {
+    func showErrorAlert(for error: Error?) {
         // TODO: MXMOBILE-2491 Log analytic when receiving an eror
-        debugPrint(error)
+        debugPrint(String(describing: error))
 
         let alert = UIAlertController(
             title: nil,
             message: NSError.stp_unexpectedErrorMessage(),
             preferredStyle: .alert)
         presentPopup(alert)
     }
+
+    func cleanupDownloadedFile() {
+        guard let downloadedFile else { return }
+
+        // Delete file since we don't need it anymore
+        DispatchQueue.global(qos: .background).async { [weak self] in
```

Comment:
I'm not entirely sure we _need_ to dispatch, but go/llm indicates that it's best practice since it's synchronous and can take some time. But Apple's docs don't state a recommendation:
https://developer.apple.com/documentation/foundation/filemanager/1413590-removeitem

Reply:
Related - should we dispatch when creating the temp directory? I don't think so given that `decideDestinationUsing` is async, but interested in feedback
https://github.com/stripe/stripe-ios/blob/master/StripeConnect/StripeConnect/Source/Internal/Webview/ConnectWebView.swift#L246

Reply:
This looks right to me.